### DISCUSSION TITLE
Showcase all the information contained in the Public and Internal tags, when printing a PDF export of a case

### DISCUSSION THREAD

**Discussion by http-alba** [globaleaks/globaleaks-whistleblowing-software#3715] [Ideas]
After the development and release of the new layout for exporting and printing the cases in pdf format, we've realised that only one of the tags can be selected. In case we need to disclose all the information contained in the "public" and the "internal" tags, our tennants would need to print the case two times.

We're proposing an added feature to showcase the information contained in both tags at once and we'd also like to adapt the layout so we can try to show it in a single page as often as possible. We understand that if the notes are long that won't be possible.

**Comment by evilaliv3** [comment]
Thank you @http-alba 

I think this is a valuable feature idea. I've created this ticket in order to keep track of it: https://github.com/globaleaks/GlobaLeaks/issues/3716
